multi-project/core/src/main/java/com/epam/Utils.java

```diff
@@ -0,0 +1,16 @@
+package com.epam;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+public class Utils {
+    public static boolean isAllPositiveNumbers(String... str) {
+        if (ArrayUtils.isNotEmpty(str) && str.length > 0) {
```

Comment:
`str.length > 0` is redundant here. `ArrayUtils::sNotEmpty` already has this check inside

Reply:
done , remove redundant str.length > 0